net/qBittorrent/Portfile

```diff
@@ -36,17 +36,19 @@ compiler.cxx_standard 2014
 compiler.blacklist-append *gcc-3.* *gcc-4.* {clang < 900.0.39.2}
 
 qt5.depends_component \
-                qttools
+                qttranslations
 
 configure.args-append \
                 --disable-gui
 
 destroot {
         if {[variant_isset gui]} {
-                exec ${qt_dir}/bin/macdeployqt ${worksrcpath}/src/qBittorrent.app -codesign=- >/dev/null 2>/dev/null
+                reinplace "s|Plugins = PlugIns|Plugins = ${qt_dir}/plugins|g" ${worksrcpath}/src/qBittorrent.app/Contents/Resources/qt.conf
+                exec /usr/bin/codesign --sign - ${worksrcpath}/src/qBittorrent.app
```

Comment:
I still wonder why you're using `exec` instead of `system` like we use elsewhere. IIRC, `exec` bypasses MacPorts safeguards like sandboxing while `system` respects them. So perhaps:

```
system -W ${worksrcpath}/src "/usr/bin/codesign --sign - qBittorrent.app"
```